docs/pages/config-plugins/introduction.mdx

```diff
@@ -1,59 +1,62 @@
 ---
-title: 'Config plugins: Introduction'
-description: An introduction to config plugins for the Expo project.
+title: Introduction to config plugins
+description: An introduction Expo config plugins.
 sidebar_title: Introduction
 ---
 
-import { Terminal } from '~/ui/components/Snippet';
+import { BookOpen02Icon } from '@expo/styleguide-icons/outline/BookOpen02Icon';
 
-An automatic setup for adding a native module to your project is possible. Sometimes, a module requires a more complex setup. A config plugin can be used to automatically configure your native project for a module and reduce the complexity by avoiding interaction with the native project.
+import { BoxLink } from '~/ui/components/BoxLink';
+
+When using a [Continuous Native Generation (CNG)](/workflow/continuous-native-generation/) project, you might want to implement native project (**android** and **ios** directories) changes without directly interacting with the native project files. In such scenarios, you can use a config plugin to automatically configure your native project.
```

Comment:
```suggestion
When using [Continuous Native Generation (CNG)](/workflow/continuous-native-generation/) in a project, native project (**android** and **ios** directories) changes are implemented without directly interacting with the native project files. Instead, you can use config plugin to automatically configure your native project beyond what can be configured using the default app config props.
```
CNG uses config plugins basically by definition. There is no "may", only "will" :-).

Reply:
maybe note about default props is redundant given below